Raspi/Test/mock_wireless_module.py

```diff
@@ -0,0 +1,170 @@
+# MQTT wireless sensor test script that sends fake randomly generated data
+# To do a single test --> python3 mqtt_wireless_test.py -t 0
+
+import sys
+import time
+import random
+import argparse
+import json
+import paho.mqtt.client as mqtt
+
+class Sensors:
+
+    decimals = 2            # number of decimal places for the random generated val
+    percentRange = 0.05     # percent range for the rangomly generated value
+
+    def CO2(val=325):
+        # value in ppm (normal background conentration 250-400ppm)
+        return round(val + val*random.uniform(-Sensors.percentRange, Sensors.percentRange), Sensors.decimals)
+
+    def humidity(val=75):
+        # value as a percentage of humidity
+        return round(val + val*random.uniform(-Sensors.percentRange, Sensors.percentRange), Sensors.decimals)
```

Comment:
Would be good to split this up!

``` python
def humidity(val=75):
        random_humidity = val + val*random.uniform(-Sensors.percentRange, Sensors.percentRange)
        return round(random_humidity, Sensors.decimals)
```
Like all humans, we're all lazy. If I were to skim this code, I would be able to read what is returned from this function. It also means I don't have t write unnecessary comments to the code too as the code itself its verbose already.

Reply:
This line `round(val + val*random.uniform(-Sensors.percentRange, Sensors.percentRange), Sensors.decimals)` is repeated a lot. It might be a good idea to refactor this into its own function.

Reply:
Yeah I found that as well, now it is just one function that gets called every time